package.json

```diff
@@ -9,11 +9,10 @@
     "@babel/polyfill": "^7.0.0",
     "@bigcommerce/stencil-utils": "^4.1.1",
     "creditcards": "^3.0.1",
-    "easyzoom": "^2.5.0",
+    "easyzoom": "^2.5.2",
     "foundation-sites": "^5.5.3",
     "jquery": "^3.4.1",
     "jstree": "vakata/jstree",
-    "lazysizes": "4.1.2",
```

Comment:
Is there a reason why we no longer want to pull this dependency as a npm package?

Reply:
It's because I wanted to move lazysizes to execute before head.scripts:

https://github.com/bigcommerce/cornerstone/pull/1507#discussion_r289417748

This improves perceptual performance of the page, especially when there are things in head.scripts that are either blocking or simply hog bandwidth that could go to downloading the bundle (therefore they are _effectively_ blocking on a slow connection).

Reply:
I see, you want to include `lazysizes` separately (from the main bundle / chunks). Perhaps you just need to specify a separate `entry` in your Webpack config to produce a separate file for that dependency. One of the benefits about keeping it as a `npm` dependency is that we can easily keep track of the version we're using and if there's security vulnerability with that version, Dependabot can warn us about it and suggest an automated upgrade.

Reply:
Thanks! I'll look into this.